app/models/Item.scala

```diff
@@ -214,6 +214,10 @@ case class Item(id: Int,            // DB key
                   </epdcx:statement>
                 }
 
+                <epdcx:statement epdcx:propertyURI="http://purl.org/dc/elements/1.1/source">
```

Comment:
@richardrodgers can you confirm this is what we decided on? Thanks for the discussion and help determining a good choice for our needs here.